app/lib/models/collection.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const Api = require(`${paths.lib.models}/api`)
+const CollectionRoutes = require(`${paths.lib.helpers}/collection-routes`)
+const config = require(paths.config)
+
+const Collection = function () {
+
+}
+
+Collection.prototype.buildCollectionRoutes = function () {
+  // new Api(config)
+  return this.getCollections()
+    .then(apiCollections => new CollectionRoutes(apiCollections))
+}
+
+Collection.prototype.getCollections = function () {
+  return Promise.all(
+    config
+      .get('apis')
+      .map(api => new Api(api)
```

Comment:
@eduardoboucas I've been in two minds about that, but tend to agree. Will change this.